includes/template.php

```diff
@@ -56,7 +56,28 @@
 $args = array( 'timeout' => 30 );
 $response = wp_remote_get( $url, $args );
 
-if ( !is_wp_error( $response ) ) {
+if ( is_wp_error( $response ) ) {
+  // If the request fails, set the status to 500.
+  status_header( 500 );
+  if ( WP_DEBUG === true ) {
+    // If the WordPress is in debug mode, throw an error.
+    throw new Exception( $response->get_error_message() );
+  } else {
+    // If not, send the error page.
+    require_once plugin_dir_path( __FILE__ ) . 'error.php';
+  }
+} else if ( substr( (string) $response[ "response" ][ "code" ], 0, 1 ) === "5" ) {
```

Comment:
Wondering if there's a better way of knowing if the error code starts with `5`... :thinking: :laughing:

Reply:
It's more than likely 😆